Dockerfiles/cluster-agent/dist/templates/header.tmpl

```diff
@@ -28,13 +28,17 @@
     {{$name}}: {{$value}}
     {{- end }}
   {{- end }}
-      hostname provider: {{.hostnameStats.provider}}
+    hostname provider: {{.hostnameStats.provider}}
+  {{- if gt (len .hostnameStats.errors) 0 }}
+    unused hostname providers:
```

Comment:
Maybe unavailable instead of unused

Reply:
sometimes the agent logic chooses not to use a hostname provider even it it's available, so I have a preference for unused